src/k2/cmd/tpcc/Log.h

```diff
@@ -0,0 +1,29 @@
+/*
+MIT License
+
+Copyright(c) 2020 Futurewei Cloud
```

Comment:
2021? Same thing for the other new files.

Reply:
done